src/sonic-dhcp-utilities/dhcp_utilities/dhcpservd/dhcp_cfggen.py

```diff
@@ -149,12 +149,13 @@ def _construct_obj_for_template(self, dhcp_server_ipv4, port_ips, hostname, cust
                                   .format(dhcp_interface_name))
                     continue
                 curr_options = {}
-                for option in dhcp_config["customized_options"]:
-                    if option in customized_options.keys():
-                        curr_options[option] = {
-                            "always_send": customized_options[option]["always_send"],
-                            "value": customized_options[option]["value"]
-                        }
+                if "customized_options" in dhcp_config:
+                    for option in dhcp_config["customized_options"]:
+                        if option in customized_options.keys():
```

Comment:
Should we add warning log when an option mentioned in `dhcp_config` but not detailed in `customized_options`?

Reply:
Added